docs/meetups/index.md

```diff
@@ -23,7 +23,7 @@ We'd be happy to promote it and list it here.
 -   [Boulder/Denver, CO, USA](http://www.meetup.com/Boulder-Denver-WriteTheDocs-Meetup/)
 -   [Dallas, TX, USA](http://www.meetup.com/Write-the-Docs-Dallas/)
 -   [Houston, TX, USA](https://www.meetup.com/Write-the-Docs-Houston/)
--   [New York, NY, USA](http://www.meetup.com/Write-The-Docs-NY/)
+-   [New York City, NY, USA](../nyc.html) ([meetup.com](https://www.meetup.com/WriteTheDocsNYC/))
```

Comment:
I'm unsure if this is how the link URL should be. While my initial though was to use `/meetups/nyc/` as the URL, it wouldn't work when locally building. Only `../nyc.html` worked.